### PR TITLE
Fixes syndicate commandos not leaving gibs

### LINKED ISSUE #22865: Syndicate commandos  dont leave a body
Warpstorm event: Buncha syndicate commandos and elite nuke ops  show up.

Elite ops drop their bodies with all their gear. Commandoes in red hardsuits just disapppear from reality

### PR DESCRIPTION
**Pull request by Iamgoofball**
Fixes #22865

### CODE REVIEW

**Comment by Haevacht** [comment]
This actually tested?

**Comment by Iamgoofball** [comment]
yes

**Comment by Jordie0608** [comment]
This makes the type identical to its parent: https://github.com/tgstation/tgstation/blob/master/code/modules/mob/living/simple_animal/hostile/syndicate.dm#L75
Which means it's kinda pointless, make it a blank type or change the portal storm spawn types to the parent and remove the `/noloot` sub.
Either way you'll also need to do this change for the ranged syndicate commando as they're also spawned by this event: https://github.com/tgstation/tgstation/blob/master/code/modules/events/portal_storm.dm#L11

**Comment by KorPhaeron** [comment]
This doesn't fix the problem

**Comment by Iamgoofball** [comment]
I just spawned a syndicate commando and he exploded into gibs

You clarified the lack of gibs was a bug in the issue report yourself

**Comment by AnturK** [comment]
Change the commit message.

File: tgstation.dme [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 // BEGIN_INCLUDE
 #include "_maps\__MAP_DEFINES.dm"
-#include "_maps\tgstation2.dm"
+#include "_maps\runtimestation.dm"
```
Comment (XDTM) [inline tgstation.dme]: forgot the map on, although i guess this definitely proves you tested

**Comment by ma44** [comment]
Just make the gibs of a body the loot instead of making them drop energy shields ya donk

**Comment by Iamgoofball** [comment]
That's literally what I did here

**Comment by Mervill** [comment]
Does this really fix the referenced 56 issues?

**Comment by MrStonedOne** [comment]
This does not fix those issues, this is goof ball trying to meme his way out of the freeze

**Comment by Iamgoofball** [comment]
all those issues are either 1. super fucking old and no longer exist, or 2. issues with a since-refactored system

**Comment by MrStonedOne** [comment]
image

**Review by Cyberboss — CHANGES_REQUESTED** [review]
> Commit message

**Comment by KorPhaeron** [comment]
Does that actually work?

**Comment by ChangelingRain** [comment]
Yes

**Comment by Iamgoofball** [comment]
you can use it on PRs too

**Comment by Jordie0608** [comment]
Hah, this would be funny enough to pass if it wasn't completely scummy.

**Pull request closed without merging**